client/src/components/CKANRegistryDisplay.js

```diff
@@ -117,25 +124,54 @@ class CKANRegistryDisplay extends Component {
    */
   getVisibleFields() {
     return this.props.fields
-      .filter(field => parseInt(field.ShowInResultsView, 10) === 1)
+      .filter(field => field.ShowInResultsView)
       .map(field => field.OriginalLabel);
   }
 
+  /**
+   * Finds all visible fields that have configured "ResultConditions", e.g. fields
+   * that should only show data in them that match certain conditions. When found,
+   * these will be applied as query filters before the data is loaded.
+   *
+   * @param {Query} query
+   */
+  applyDefaultFilters(query) {
+    const { fields } = this.props;
+
+    fields
+      .filter(field => {
+        if (!field.DisplayConditions) {
+          return false;
+        }
+        // Don't filter on columns that aren't in the displayed dataset or don't have
+        // any configured display conditions defined
+        return field.DisplayConditions.length && field.ShowInDetailView;
+      })
+      .forEach((field) => {
+        field.DisplayConditions.forEach((displayCondition) => {
+          // Add our filter statement
+          query.filter(
+            field.OriginalLabel, // column
+            displayCondition['match-text'], // term
+            true, // strict
```

Comment:
Hmmmn. I wonder if this is what's expected... Should the "Must match" configuration on a column look for exact matches @camfindlay?

Reply:
It's probably more a contains, the key use case for this is pre-filtering by some sort of category. In the case a column contains more than 1 category (eg. comma separated "Health, Youth") and let's say you wanted to create a "Youth services" register from a larger master dataset. You'd want to "Match "Youth" in the Category column. If it's an exact match, the above record wont be included even though it's a legit entry for our case "Youth != Heath, Youth". 

How is the dropdown filter working for delimiter separated terms? I'd imagine the pre-filter would work in much the same way.

Reply:
I’m thinking about it again - “must match Auckland” could also return South Auckland too. Maybe this isn’t required after all